assets/styles/cirrus.css

```diff
@@ -328,6 +328,10 @@
   -webkit-mask-image: url("../icons/doctypes/profile.svg");
   mask-image: url("../icons/doctypes/profile.svg");
 }
+.perm[class^="io-cozy-calendar"] {
+  -webkit-mask-image: url("../icons/doctypes/calendar.svg");
+  mask-image: url("../icons/doctypes/calendar.svg");
```

Comment:
Can you add the `calendar.svg` and `school.svg` icons in the `assets/icons/doctypes` directory?

Reply:
It's been added !